components/Form/FormTextArea/__tests__/index.spec.tsx

```diff
@@ -13,14 +13,24 @@ describe('FormTextArea', () => {
         label="Name"
         placeholder="Enter your name"
         value="John Doe"
-        onChange={jest.fn()}
```

Comment:
No need to have this here if we are testing it in isolation in another test. Just removed it to keep it clean.